cluster/examples/hook-sidecar-consumer.yml

```diff
@@ -0,0 +1,42 @@
+status: {}
```

Comment:
Make sure this is autogenerated by tools/vms-generator/vms-generator.go and the vms-generator.go file is updated.. I don't see any changes to vms-generator.go.

Reply:
Did not know that's a thing, thanks